lib/text.rb

```diff
@@ -0,0 +1,20 @@
+require 'twilio-ruby'
+
+class Text
+  def initialize(number, acc_sid, a_token)
```

Comment:
Not sure I understand the thinking here, particularly with the method commented out in order.rb. Would be good to chat it through